endoreg_db/views/classification_views.py

```
@@ -0,0 +1,70 @@
+# endoreg_db/views/classification_views.py
+from rest_framework.viewsets import ReadOnlyModelViewSet
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework import serializers
+from ..models import FindingLocationClassification, FindingMorphologyClassification
+
+class LocationClassificationSerializer(serializers.ModelSerializer):
+    choices = serializers.SerializerMethodField()
+
+    class Meta:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)